manifests/namespace-install/workflow-controller-rbac/workflow-controller-rolebinding.yaml

```diff
@@ -8,4 +8,18 @@ roleRef:
   name: argo-role
 subjects:
 - kind: ServiceAccount
-  name: argo
\ No newline at end of file
+  name: argo
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
interesting ... @jessesuen what do you think to allowing a namespace install to read cluster templates? I think we should not allow this